website/notifications/emails.py

```diff
@@ -60,10 +60,14 @@ def notify_mentions(event, user, node, timestamp, **context):
 def notify_global_event(event, sender_user, node, timestamp, recipients, template=None, context=None):
     event_type = utils.find_subscription_type(event)
     sent_users = []
+    if not context:
+        context = {}
 
     for recipient in recipients:
         subscriptions = get_user_subscriptions(recipient, event_type)
         context['is_creator'] = recipient == node.creator
+        if node.provider:
+            context['has_psyarxiv_chronos_text'] = node.has_permission(recipient, ADMIN) and node.provider.name == 'PsyArXiv'
```

Comment:
Will the provider ever have different casing, or is it safe to do this simple comparison?